fontir/src/orchestration.rs

```diff
@@ -137,33 +97,62 @@ impl Context {
             .unwrap();
         let buf_io = BufWriter::new(raw_file);
         serde_yaml::to_writer(buf_io, &content)
-            .map_err(|e| panic!("Unable to serialize to {:?}: {}", file, e))
+            .map_err(|e| panic!("Unable to serialize\n{:#?}\nto {:?}: {}", content, file, e))
```

Comment:
ultimately we're going to want to actually return errors for things like this. I'm not sure if it's worth doing now, but it's worth keeping in mind...

Reply:
https://github.com/googlefonts/fontmake-rs/issues/84 as I think we have a number of places where we panic or unwrap things that could legitimately fail.